migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,314 @@
+-- Enums
+
+CREATE TYPE gender                  AS ENUM ('male', 'female');
+CREATE TYPE role                    AS ENUM ('user', 'admin');
+CREATE TYPE status                  AS ENUM ('available', 'unavailable');
+CREATE TYPE association             AS ENUM ('sponsor', 'organization', 'media');
```

Comment:
Might be worth to add 'other'. You never know what weird associations can come up.